packages/cocoindex/tests/test_proxy_manager.py

```
@@ -79,6 +80,57 @@ def test_no_derivation_without_project_dir(self) -> None:
+        alias_project = alias_root / "project-link"
+        alias_project.symlink_to(real_project, target_is_directory=True)
+
+        result_real = proxy_mgr.get_proxy_config(config, project_dir=str(real_project))
+        result_alias = proxy_mgr.get_proxy_config(config, project_dir=str(alias_project))
+
+        assert result_real["port"] == result_alias["port"]
+
+
+class TestBuildProxyUrl:
+    def test_claude_uses_sse(self) -> None:
+        url = proxy_mgr.build_proxy_url("claude", SAMPLE_CONFIG, "/tmp/project")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/project"

(S108)